helios-services/src/main/java/com/spotify/helios/servicescommon/statistics/NoopZooKeeperMetrics.java

```diff
@@ -17,7 +17,14 @@
 
 package com.spotify.helios.servicescommon.statistics;
 
+import com.codahale.metrics.Timer;
+
 public class NoopZooKeeperMetrics implements ZooKeeperMetrics {
   @Override
   public void zookeeperTransientError() {}
+
+  @Override
+  public Timer timer(String name) {
+    return new Timer();
```

Comment:
If you changed this interface slightly you could do an actual no-op implementation. E.g. something like:

```
void updateTimer(String name, final long durationMillis);
```